src/Faker/Provider/pl_PL/Payment.php

```diff
@@ -5,81 +5,76 @@
 class Payment extends \Faker\Provider\Payment
 {
     /**
-     * @var array list of Polish banks, source: http://www.nbp.pl/banki_w_polsce/ewidencja/ewidencja.html
+     * @var array list of Polish banks, source: https://ewib.nbp.pl/
```

Comment:
Does this dataset comply with the MIT license of this package? Or is it copyrighted?

Reply:
I think this is free for OS projects. I already send an e-mail to NBP with your question